less/index.less

```diff
@@ -4,6 +4,23 @@
    License: none (public domain)
 */
 
+// Variables
+
+@Maingray: #A9A9A9;
```

Comment:
Great job commenting code.  Makes life easier for others reading it.  Good use of a variable.